### ISSUE TITLE
Support modern color function notation

### ISSUE THREAD

**Issue by daanvosdewael** [EdMSL/postcss-color-converter#18]
When using the plug-in with modern color function notation, the conversion is not correct. An example with `rgb()` to `hsl()` with alpha:

```css
// Input
body {
  background-color: rgb(255 255 255 / 0);
}

// Output
body {
  background-color: hsla(60, 100%, 50%, 1);
}

// Expected
body {
  background-color: hsl(0 0% 100% / 0);
}
```

More info about the notation can be found in the spec: https://drafts.csswg.org/css-color/#the-hsl-notation

**Comment by EdMSL** [comment]
I'll see what I can do.

**Comment by EdMSL** [comment]
Fixed in 1.1.1.
